### PR TITLE
CHE-995: mark architecture field as beta

### PR DESCRIPTION
**Pull request by garagatyi**
@skabashnyuk @evoevodin Please review

### CODE REVIEW

File: core/che-core-api-model/src/main/java/org/eclipse/che/api/core/model/machine/MachineConfig.java [hunk 1]
```diff
@@ -60,6 +60,9 @@
 
     /**
      * Architecture of target machine. Default is 'linux_amd64'.
+     *
+     * <p/>This method is in beta. And can be removed in next versions.
```
Comment (sleshchenko) [inline core/che-core-api-model/src/main/java/org/eclipse/che/api/core/model/machine/MachineConfig.java:66]: com.google.common.annotations.Beta ?

Comment (garagatyi) [inline core/che-core-api-model/src/main/java/org/eclipse/che/api/core/model/machine/MachineConfig.java:66]: done

**Comment by skabashnyuk** [comment]
ok

**Comment by sleshchenko** [comment]
OK

**Pull request merged**